docs/blog/august-contributions-showcase.md

```diff
@@ -0,0 +1,134 @@
+
+fdir is a very fast and lightweight library for traversing the file system and has optional glob support.
+
+Already a great contributor in the e18e space, [@superchupudev](https://x.com/superchupu) has also provided us with [tinyglobby](https://github.com/SuperchupuDev/tinyglobby).
+
+tinyglobby adds a useful layer on top of fdir for when you still need a glob interface and consistent behaviour with other glob libraries:
+
+```ts
+import { glob } from 'tinyglobby'
+
+const files = await glob(['src/*.ts'])
+```
+
+So many projects have been quick to adopt these libraries, saving on install size and getting great performance gains at the same time. To name a few:
+
+- [nuxt](https://github.com/nuxt/nuxt)
+- [vitest](https://github.com/vitest-dev/vitest/)
+- [vitepress](https://github.com/unocss/unocss)
+- [unocss](https://github.com/unocss/unocss)
+- [pkg-pr-new](https://github.com/stackblitz-labs/pkg.pr.new)
+- [tsup](https://github.com/egoist/tsup)
+- and many, many more
+
+Great work by both [@thecodrr](https://x.com/thecodrr) and [@superchupudev](https://x.com/superchupu) here!
+
+# package-manager-detector
+
+[`package-manager-detector`](https://github.com/antfu-collective/package-manager-detector) is used to detect the local package manager (e.g. `npm`).
+
+Much lighter than other alternatives, this package has quickly been adopted by many projects across the ecosystem.
+
+As an example, [@benjaminmccann](https://x.com/benjaminmccann) switched from `preferred-pm` (1MB) to `package-manager-detector` (28KB) in [changesets](https://github.com/changesets/changesets/pull/1446) recently. Great work :pray:
+
+Thanks to [@antfu7](https://x.com/antfu7) and [@userquin](https://x.com/userquin) for building this!
+
+# tschema
+
+[tschema](https://github.com/lukeed/tschema) is a cool new library to generate JSON schema types by [@lukeed05](https://x.com/lukeed05).
+
+Super light and fast, this is another package quickly being adopted in the e18e space.
+
+```ts
+import * as t from 'tschema'
+
+const User = t.object({
+  uid: t.integer()
+})
+```
+
+# knip
+
+[knip](https://github.com/webpro-nl/knip) is a tool for detecting unused files and dependencies in your project. This has been incredibly useful and widely used across the e18e community already.
+
+For example, [mocha](https://github.com/mochajs/mocha/pull/5042) has already adopted this and cleaned up a huge amount of deep dependencies thanks to the efforts of [@voxpelli](https://x.com/voxpelli).
+
+# dotenvx
+
+The maintainers of [dotenvx](https://github.com/dotenvx/dotenvx) have been hard at work with [@superchupudev](https://x.com/superchupu) to massively reduce the install size and depedency tree complexity.
+
+Down from [142 dependencies](https://npmgraph.js.org/?q=@dotenvx/dotenvx@1.0.0) to only [28 dependencies](https://npmgraph.js.org/?q=@dotenvx/dotenvx), this is a great achievement :tada:
+
```

Comment:
payload cms could be another good one to mention
https://npmgraph.js.org/?q=payload - 579 dependencies
https://npmgraph.js.org/?q=payload@3.0.0-beta.100 - 132 dependencies

the payload team did all the work on that one. Some of its dependencies like bcherny's `json-schema-to-typescript` were also cleaned up

Reply:
Will get it added 👍